UnicornRender/video/include/unicorn/video/Material.hpp

```diff
@@ -87,10 +87,54 @@ class Material
     /** @brief Returns color */
     UNICORN_EXPORT glm::vec3 GetColor() const;
 
+    /**
+     * @brief Sets texture cordinates
+     *
+     * @note Can't set texture coordinates without texture.
+     *
+     * @param x texture offset by x in pixels
+     * @param y texture offset by y in pixels
+     * @param width offset width in pixels
+     * @param height offset height in pixels
+     */
+    UNICORN_EXPORT void SetTextureCoordinates(int32_t x, int32_t y, int32_t width, int32_t height);
+
+    /**
+     * @brief Sets normalized texture cordinates
+     *
+     * @note Can't set texture coordinates without texture.
+     *
+     * @param x texture offset by x from 0.0 to 1.0
+     * @param y texture offset by y from 0.0 to 1.0
+     * @param width offset width from 0.0 to 1.0
+     * @param height offset height from 0.0 to 1.0
+    */
+    UNICORN_EXPORT void SetNormalizedTextureCoordinates(float x, float y, float width, float height);
+
+    /** @brief Drops previous set texture coordinates */
```

Comment:
`Resets texture coordinates to default values`? And list values in the doxygen comment